lib/collage.php

```diff
@@ -22,6 +22,14 @@ function createCollage($srcImagePaths, $destImagePath) {
 
         $tempSubImage = imagecreatefromjpeg($srcImagePaths[$i]);
 
+        if ($takeFrame) {
```

Comment:
Is this not the same code as in `api/applyEffects.php`?

Reply:
Didn't got it working from a central point...

Reply:
Seems like this is left. There's some duplicate code to apply a frame which maybe you or someone else is able to centralize? I tried it but failed on that... :disappointed:

Reply:
can be optimized in future